index.html

```diff
@@ -29,352 +32,374 @@
+              <img src="images/desktop-section-1.png" alt="" />
             </div>
-  
-  </div>
-        <!---End Mark up for s2 Desktop -->
-        
-    <!------End of styling for Desktop card 2-->
-        <div class="s2-container">
-          <div class="image-container2"></div>
+            <div class="card-text">
+              <h2>Tonic</h2>
+              <div class="list-container">
+                <ul>
+                  <li><a href="" class="text">CANOPY</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-2">Back End Dev</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-3">2015</a></li>
+                </ul>
+              </div>
 
-          <h1 class="newh2">Multi-Post-Stories</h1>
-          <div class="list-container">
-            <ul>
-              <li><a href="" class="text">CANOPY</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-2">Back End Dev</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-3">2015</a></li>
-            </ul>
-          </div>
-          <div>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          </div>
-          <div class="base-links flex-card">
-            <ul>
-              <li><a href="" class="base-link-text1">html</a></li>
-              <li><a href="" class="base-link-text2">css</a></li>
-              <li><a href="" class="base-link-text3">javascript</a></li>
-            </ul>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="base-links">
+                <ul>
+                  <li><a href="" class="base-link-text1">html</a></li>
+                  <li><a href="" class="base-link-text2">css</a></li>
+                  <li><a href="" class="base-link-text3">javascript</a></li>
+                </ul>
+              </div>
+              <div class="base-button">
+                <a href="#">See Project</a>
+              </div>
+            </div>
           </div>
-          <div class="base-button flex-card">
-            <a href="">See Project</a>
+          <!---End of desktop card 1 Mark up-->
+          <div class="s1-container">
+            <div class="image-container"></div>
+            <div class="content">
+              <h2>Tonic</h2>
+              <div class="list-container">
+                <ul>
+                  <li><a href="" class="text">CANOPY</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-2">Back End Dev</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
```

Comment:
- [ ]  I noticed here the alt is empty,, kindly add a text describing the image for accessibility issues 🙏🏻